### PR TITLE
chore: (TNLT-6786) BREAKING CHANGE - Updates ReactNative to 0.63.4 and reverts Android Dimensions

### PR DESCRIPTION
**Pull request by Geoff-Ford**
https://nidigitalsolutions.jira.com/browse/TNLT-6786

* Updates TCN to use RN 0.63.4 as this version contains the fix for Dimensions that prevented Android from responding correctly to orientation changes.
* Removes all references to the temporary fix for Android where we received dimensions via initProps to the various pages and set listeners for changes to them.

I have locally built both iOS and Android with the relevant changes to those repos which I am putting up in separate PRs once this is release and I have a new TCN version number to add to those PRs.

### CODE REVIEW

File: packages/utils/src/dimensions-util.js [hunk 1]
```diff
@@ -18,28 +7,10 @@ export const getDimensions = (
 ) => ({ width, height, fontScale });
 
 export const addDimensionsListener = (type, handler) => {
-  if (Platform.OS === "android") {
-    const wrappedHandler = (data) => {
-      return handler({
-        window: {
-          width: data.displayWidth,
```
Comment (kamiranoff) [inline packages/utils/src/dimensions-util.js:25]: One question. Are these properties not used anymore? Do I need them in my own PR?

**Review by kamiranoff — APPROVED** [review]

**Review by aronshamash — APPROVED** [review]

File: packages/utils/src/dimensions-util.js (hunk 1 above)
Comment (Geoff-Ford) [reply packages/utils/src/dimensions-util.js:25]: As discussed offline, these properties are no longer used. We get them from RN Dimensions as used to originally before the Dimensions issues on Android.
